### PR TITLE
Issue #13213: Remove '//ok' comments from annotationonsameline

### PR DESCRIPTION
**Pull request by Rohanraj123**
part of #13213 

Link to check documentation : [click here](https://checkstyle.sourceforge.io/checks/annotation/annotationonsameline.html#AnnotationOnSameLine)

proof that all suppressions are removed : 

```
rajro@DESKTOP-HSV42DL MINGW64 ~/checkstyle (master)
$ grep annotationonsameline config/checkstyle-input-suppressions.xml
            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineRecordsAndCompactCtors.java"/>
            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineRecordsAndCompactCtors.java"/>
            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck.java"/>
            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck.java"/>
            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck2.java"/>
            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck2.java"/>
            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck3.java"/>
            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck3.java"/>

rajro@DESKTOP-HSV42DL MINGW64 ~/checkstyle (master)
$ git checkout issue-13213-annotationonsameline
Switched to branch 'issue-13213-annotationonsameline'

rajro@DESKTOP-HSV42DL MINGW64 ~/checkstyle (issue-13213-annotationonsameline)
$ grep annotationonsameline config/checkstyle-input-suppressions.xml
            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineRecordsAndCompactCtors.java"/>
            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineRecordsAndCompactCtors.java"/>

rajro@DESKTOP-HSV42DL MINGW64 ~/checkstyle (issue-13213-annotationonsameline)
$ echo $?
0


```

### CODE REVIEW

**Review by romani — APPROVED** [review]
Ok to merge

File: config/checkstyle-input-suppressions.xml [hunk 1]
```diff
@@ -1605,18 +1605,6 @@
...
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck2.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck2.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck3.java"/>
-  <suppress id="UnnecessaryOkComment"
-            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck3.java"/>
```
Comment (nrmancuso) [inline config/checkstyle-input-suppressions.xml:1619]: >proof that all suppressions are removed :

Please explain why `grep` shows two results and has an exit code of zero if all suppressions are removed.

**Review by nrmancuso — CHANGES_REQUESTED** [review]
Please resolve conflicts.

Item:

**Comment by Rohanraj123** [comment]
```
rajro@DESKTOP-HSV42DL MINGW64 ~/checkstyle (master)
$ grep annotationonsameline config/checkstyle-input-suppressions.xml
            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineRecordsAndCompactCtors.java"/>
            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineRecordsAndCompactCtors.java"/>
            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck.java"/>
            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck.java"/>
            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck2.java"/>
            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck2.java"/>
            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck3.java"/>
            files="checks[\\/]annotation[\\/]annotationonsameline[\\/]InputAnnotationOnSameLineCheck3.java"/>

rajro@DESKTOP-HSV42DL MINGW64 ~/checkstyle (master)
$ git checkout issue-13213-annotationonsameline
Switched to branch 'issue-13213-annotationonsameline'

rajro@DESKTOP-HSV42DL MINGW64 ~/checkstyle (issue-13213-annotationonsameline)
$ grep annotationonsameline config/checkstyle-input-suppressions.xml

rajro@DESKTOP-HSV42DL MINGW64 ~/checkstyle (issue-13213-annotationonsameline)
$ echo $?
1
```

Required grep check..

**Comment by romani** [comment]
@Rohanraj123, keep single commit always

**Comment by Rohanraj123** [comment]
> @Rohanraj123, keep single commit always

I read that in guidelines, I did this by mistake and now I am unable to remove it . Can you please suggest me something to do that. I made the same change in #13690 correctly but here I made some mistake and now I dont know how to remove this unwanted commit.

**Comment by nrmancuso** [comment]
@Rohanraj123 `mvn clean verify` should always be passing on your local before pushing changes to Github

**Comment by Rohanraj123** [comment]
I am not able to remove the extra unwanted commit. How can i do that ? Can you please suggest me something @nrmancuso

**Comment by romani** [comment]
Remove branch on your local and recreate it from scratch, push with force back to same branch name to GitHub

**Comment by romani** [comment]
Resolve conflict please

**Comment by Rohanraj123** [comment]
> Resolve conflict please

Can you please suggest me how can I resolve this conflit. I research a lot and applied certain approaches but it didn't worked. @romani

**Comment by romani** [comment]
do backup of your repo to some folder on your local drive
in git repo:
```
git checkout master
git pull upstream master
git branch -D issue-13213-annotationonsameline
git checkout -b issue-13213-annotationonsameline
```
copy all files except for ...supression.xml
apply update to suppression.xml manually
```
git add .
git commit -m ....
git push origin issue-13213-annotationonsameline
```

**Pull request closed without merging**